langchain4j-core/src/main/java/dev/langchain4j/model/chat/request/ToolMode.java

```diff
@@ -0,0 +1,20 @@
+package dev.langchain4j.model.chat.request;
+
+/**
+ * TODO
+ */
+public enum ToolMode {
+    // TODO name: ToolChoice? FunctionMode?
```

Comment:
Maybe even more verbose like `ToolChoiceMode`?

Reply:
I’m not a fan of either ToolMode or ToolChoice as is not really clear what it means, and ToolChoiceMode seems even less intuitive to me.

I guess we will have to pick `ToolChoice` as it is used by most LLM providers